src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private static final double AREA_COEFFICIENT = 0.5;
+    private final double firstLeg;
+    private final double secondLeg;
+    private final double hypotenuse;
+
+    public RightTriangle(String colour, double firstLeg, double secondLeg) {
+        super(colour);
+        this.firstLeg = roundValues(firstLeg);
+        this.secondLeg = roundValues(secondLeg);
+        this.hypotenuse = countHypotenuse();
```

Comment:
Inside the constructor, there should be no other logic except the field initialization. Let's round the values in the draw method if you need it